apps/console/src/extensions/configs/common.tsx

```diff
@@ -28,7 +28,8 @@ export const commonConfig: CommonConfig = {
     checkForUIResourceScopes: false,
     enableOrganizationAssociations: false,
     header: {
-        getHeaderExtensions: (): Promise<HeaderExtension[]> => Promise.resolve([]),
+        getHeaderExtensions: (_tenantDomain: string,
+            _associatedTenants: any[]): Promise<HeaderExtension[]> => Promise.resolve([]),
```

Comment:
Shall we format like follows,
```suggestion
        getHeaderExtensions: (
            _tenantDomain: string,
            _associatedTenants: any[]
        ): Promise<HeaderExtension[]> => Promise.resolve([]),
```

Reply:
Fixed with b5a3098